plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacet.java

```diff
@@ -5,7 +5,7 @@
   public static final String ID = "com.google.cloud.tools.eclipse.appengine.facet";
 
   static final String VERSION = "1";
-  static final String DEFAULT_RUNTIME_ID = "com.google.cloud.tools.eclipse.runtime.apptools";
+  static final String DEFAULT_RUNTIME_ID = "com.google.appengine.standard.runtime";
```

Comment:
Getting rid of apptools is fine, but I'd prefer to keep the package name closer. maybe com.google.cloud.tools.eclipse.appengine.standard?